cms/djangoapps/contentstore/views/helpers.py

```diff
@@ -251,6 +252,11 @@ def create_xblock(parent_locator, user, category, display_name, boilerplate=None
         # if we add one then we need to also add it to the policy information (i.e. metadata)
         # we should remove this once we can break this reference from the course to static tabs
         if category == 'static_tab':
+
+            tags = [ "action:{}".format(unicode(dest_usage_key.course_key)) ]
```

Comment:
After looking at the other datadog counters, I assumed `action` was a standard. I'll change it.